addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -266,22 +266,27 @@ func _get_history(buffer: Dictionary, tick: int) -> Dictionary:
 
 	return buffer[before]
 
+func _sanitize_by_authority(snapshot: Dictionary, sender: int) -> Dictionary:
+	var sanitized := {}
+	
+	for property in snapshot:
+		var property_entry := _property_cache.get_entry(property)
+		var value = snapshot[property]
+		var authority := property_entry.node.get_multiplayer_authority()
+		
+		if authority == sender:
```

Comment:
I think it is reasonable to expect from people working with netcode to know the concept of authority in this context, or to look it up :slightly_smiling_face: 

Also I don't see the benefit of knowing the type of these variables. Reading out the condition, "if the authority is the sender" makes sense without types.

Reply:
personally, if I see any variable operation (e.g. + or ==) without knowing the type, I am not certain what I read is correct (and have to spend time to look it up, in this case ctrl+click `get_multiplayer_authority` and then scroll up to parameters for sender variable type), because after all, authority could be some custom class, or have an operator overload.

Having offered the benefit of knowing the type of these variables, I hope that adding `_id` to these 2 variables, will be more clear to the reader, since no explicit type declaration offers ambiguity